mypy/stubgenc.py

```diff
@@ -476,25 +478,39 @@ def infer_prop_type(docstr: str | None) -> str | None:
         else:
             return None
 
-    inferred = infer_prop_type(getattr(obj, "__doc__", None))
+    docstr: str | None = getattr(obj, "__doc__", None)
+    inferred = infer_prop_type(docstr)
     if not inferred:
         fget = getattr(obj, "fget", None)
         inferred = infer_prop_type(getattr(fget, "__doc__", None))
+        docstr = getattr(fget, "__doc__", None) or docstr
     if not inferred:
         inferred = "Any"
 
     if module is not None and imports is not None and known_modules is not None:
         inferred = strip_or_import(inferred, module, known_modules, imports)
 
+    if include_docstrings and docstr:
+        # Quoted docstr
+        docstr = mypy.util.quote_docstring(docstr.strip())
+        # Indented docstr
+        docstr = "\n    ".join(docstr.split("\n"))
+
     if is_static_property(obj):
         trailing_comment = "  # read-only" if readonly else ""
         static_properties.append(f"{name}: ClassVar[{inferred}] = ...{trailing_comment}")
+        if include_docstrings and docstr:
+            static_properties.extend(f"{docstr}".split("\n"))
     else:  # regular property
         if readonly:
             ro_properties.append("@property")
             ro_properties.append(f"def {name}(self) -> {inferred}: ...")
+            if include_docstrings and docstr:
+                ro_properties.extend(f"    {docstr}".split("\n"))
```

Comment:
Did you mean to indent _after_ splitting?

Reply:
Yes, right.

`
def my_func(self):
    '''docstrings here'''
`